static/scss/ecommerce-admin.scss

```diff
@@ -74,4 +117,9 @@
     color: #fff;
     padding: 15px;
   }
+
+  .small-text {
+    font-size: 14px;
+    color: rgb(76 76 5);
```

Comment:
We should use default red which is used site-wide.
<img width="1739" alt="Screenshot 2024-10-02 at 6 10 09 PM" src="https://github.com/user-attachments/assets/cedf27c8-0df7-4e99-8fb4-e28ca26da726">